.github/actions/regression-tests/action.yml

```diff
@@ -46,7 +49,14 @@ runs:
         export DRAGONFLY_PATH="${GITHUB_WORKSPACE}/${{inputs.build-folder-name}}/${{inputs.dfly-executable}}"
         export UBSAN_OPTIONS=print_stacktrace=1:halt_on_error=1 # to crash on errors
 
-        timeout 50m pytest -m "${{inputs.filter}}" --durations=10 --timeout=300 --color=yes --json-report --json-report-file=report.json dragonfly --log-cli-level=INFO || code=$?
+        if [[ "${{inputs.epoll}}" == 'true'  ]]; then
+          # Run only replication tests with epoll
```

Comment:
why not all regression tests?

Reply:
I don't mind, I thought you wanted only the replication ones. Also we can run both `debug and release` (now I only run them in release)

Reply:
also we got failures https://github.com/dragonflydb/dragonfly/actions/runs/12669695232/job/35307742465 but I guess these are expected

Reply:
Lets run all regression tests not only replication.
Not sure they are expected to fail as Roman said he did some fixes. @romange do you want to check this or should we?

Reply:
lets skip ipv6 test for epoll. Please check  why test_big_containers  fail